aladdin-container.sh

```diff
@@ -118,6 +118,16 @@ function environment_init() {
 
 }
 
+function _helm_init() {
+    if $RBAC_ENABLED; then
+        kubectl -n kube-system create serviceaccount tiller || true
+        kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller || true
+        helm init --service-account=tiller
+    else
+        helm init
```

Comment:
why is the fallback not the same as the original code?  IE, is there a reason this isn't `helm init --upgrade --wait || true` ?

Reply:
haha whoops. good catch